Contact/Repository/ContactRepository.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    public class ContactRepository : IContactRepository
```

Comment:
I don't get the Http verb methods... Personally my interface would look something like:
Save(Contact contact) // add or update based upon whether ContactId is populated.
GetAll()
GetById(int contactId)
Delete(int contactId)

Reply:
I will be updating this. I'm not overly attached to the http verbs. I borrowed some of the controller stuff that I knew worked from a previous iteration in order to ensure that everything would function.

Reply:
I have switched to Save, Get, and Delete function names.